lib/rules/color-named/generateColorFuncs.js

```diff
@@ -152,7 +148,7 @@ function generateColorFuncs(hexString) {
 	const rgb = [0, 0, 0];
 
 	for (let i = 0; i < 3; i += 1) {
-		rgb[i] = parseInt(hexString.substr(2 * i + 1, 2), 16);
+		rgb[i] = Number.parseInt(hexString.substr(2 * i + 1, 2), 16);
```

Comment:
Why not just keep `parseInt`? [Number.parseInt vs parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt#Number.parseInt_vs_parseInt).

Reply:
>and is part of ECMAScript 2015 (its purpose is modularization of globals). Please see parseInt() for more detail and examples.